pbrd/pbr_map.c

```diff
@@ -114,6 +118,26 @@ static bool pbrms_is_installed(const struct pbr_map_sequence *pbrms,
 	return false;
 }
 
+void pbr_set_match_clause_for_pcp(struct pbr_map_sequence *pbrms, uint8_t pcp)
+{
+	if (pbrms) {
+		pbrms->match_pcp = pcp;
+		zlog_info("setting pbrms->match_pcp = %u ", pbrms->match_pcp);
+
+		pbr_map_check(pbrms, true);
+	}
+}
+
+void pbr_set_match_clause_for_vlan(struct pbr_map_sequence *pbrms,
+				   uint16_t vlan_id, uint16_t vlan_flags)
+{
+	if (pbrms) {
+		pbrms->match_vlan_id = vlan_id;
+		pbrms->match_vlan_flags = vlan_flags;
+		pbr_map_check(pbrms, true);
+	}
+}
+
```

Comment:
Same concern as above for PCP: How does the code distinguish between "match VLAN_ID==0" and "ignore VLAN_ID" ?